src/app/Instance/BridgeOverview/BOStoriesHelper.ts

```diff
@@ -12,6 +12,7 @@ export const processorData = [
     status: ManagedResourceStatus.Ready,
     flows: [],
     owner: "",
+    url: `/instance/${instanceId}/processor/${"a72fb8e7-162b-4ae8-9672-f9f5b86fb3d7"}`,
```

Comment:
I don't particularly like this approach. `url` is not a property of `ProcessorResponse`, so adding it here could be misleading.